roles/static_inventory/defaults/main.yml

```diff
@@ -19,3 +19,6 @@ private_ssh_key: ~/.ssh/openshift
 
 # The patch to store the generated config to access bastion/hosts
 ssh_config_path: /tmp/ssh.config.ansible
+
+openstack_region: regionOne
```

Comment:
To be clear, using os_client_config will return whatever you have in the `OS_REGION_NAME` environment variable or `region` in your clouds.yaml.